packages/ckeditor5-core/src/editor/editor.ts

```diff
@@ -542,6 +542,13 @@ export default abstract class Editor extends /* #__PURE__ */ ObservableMixin() {
 				} );
 			}
 
+			if ( licensePayload.licenseType === 'development' ) {
+				console.info(
+					'You are using the development version of CKEditor 5. ' +
+					'Make sure you will not use it in the production environment.'
+				);
+			}
```

Comment:
I thought about extracting logging into a separate if block for all types. But, after the change those type differ, so it's probably better this way.